### PR TITLE
add changes to include clusterName in the queries

### PR DESCRIPTION
**Pull request by khansaad**
This PR fixes the clusterName issue as part of the epic #928 

- Updated existing SQL queries to include cluster name

### CODE REVIEW

**Review by msvinaykumar — APPROVED** [review]
LGTM

File: src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java [hunk 1]
```diff
@@ -60,6 +61,8 @@
 public class UpdateRecommendations extends HttpServlet {
     private static final long serialVersionUID = 1L;
     private static final Logger LOGGER = LoggerFactory.getLogger(UpdateRecommendations.class);
+    ConcurrentHashMap<String, KruizeObject> mainKruizeExperimentMAP = new ConcurrentHashMap<>();
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java:64]: This needs to be a local hoshmap

Comment (khansaad) [reply src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java:64]: This is fixed in #937

**Review by dinogun — APPROVED** [review]
Some of the fixes are in #937

**Pull request merged**